napari_allencell_segmenter/widgets/workflow_thumbnails.py

```diff
@@ -101,13 +101,18 @@ def _add_buttons(self, workflows: List[WorkflowDefinition]):
             # TODO?: convert all images to RBGA
             pre: np.ndarray = workflow.thumbnail_pre
             post: np.ndarray = workflow.thumbnail_post
+
+            # TODO: We need a way to a) track dimension order if present in metadata and b) try to guess dimension order
+            color_channel_size: int = min(np.shape(workflow.thumbnail_pre))
+            min_index: int = np.shape(workflow.thumbnail_pre).index(color_channel_size)
```

Comment:
I'm confused about how this works. Could you elaborate?

Reply:
This is a way to guess the color channel dimension (its usually the smallest) since we dont have access to that information in this module

Reply:
Oh ok I see now, I think it would be great to have a really clear comment here like `# Assume that the color channel has the smallest dimension` in addition to the TODO comment. Same in napari_allencell_segmenter/view/workflow_steps_view.py